internal/stack/_static/logstash.conf.tmpl

```diff
@@ -12,7 +12,7 @@ input {
 filter {
   elastic_integration {
     remove_field => ['@version']
-    hosts => ["https://elasticsearch:9200"]
+    hosts => [{{ fact "elasticsearch_host" }}]
     username => {{ fact "username" }}
     password => {{ fact "password" }}
     ssl_enabled => true
```

Comment:
In this section `ssl_verification_mode` is set to `false`, should it be added here some other value?

Reply:
yeah.. removed this